x/rollapp/types/message_update_rollapp.go

```diff
@@ -85,7 +87,7 @@ func (msg *MsgUpdateRollappInformation) ValidateBasic() error {
 }
 
 func (msg *MsgUpdateRollappInformation) UpdatingImmutableValues() bool {
-	return msg.InitialSequencer != ""
+	return msg.InitialSequencer != "" || 0 < msg.MinSequencerBond
```

Comment:
rollapp min bond can't be changed post launch?

Reply:
that's a great question.

produce wise I am still not convinced what's the way to go so prefer to keep it more conservative.

I'd say that letting them change it after launch have a few caveats:
1. edge cases where we break the current assumption that bond doesn't change after launch (not sure what bugs it can introduce)

2. a lot of power to the rollapp creator **after** launch where he can e.g dos the rollapp (Change it to so high at some point where no one can bond) or alternatively want to cheat so reduce the bond to some non proportinal amount where it's worth it's while.

The gist is - as a user it's easier to agree on the "terms" of the game before you opt-in.

future plans we can make it dynamic but imo requires more thought so wouldn't wanna venture into it currently.